### ISSUE TITLE
Accessibility - "Log Out" not visible at 200% zoom

### ISSUE THREAD

**Issue by kaitlinnewson** [IQSS/dataverse#8135]
Per [WCAG 1.4.4 - Resize text](https://www.w3.org/TR/WCAG21/#resize-text), all functionality should be available at up to 200% zoom. In my testing so far, most functionality in DV is working well, but one issue I noticed is that the "Log out" menu item is not visible at this zoom level.

**What steps does it take to reproduce the issue?**

Log into Dataverse, zoom the browser to 200%, then try to access the "Log Out" menu item.

* When does this issue occur?

When the browser is zoomed to 200% (tested in Chrome and Firefox on Mac OS).

* Which page(s) does it occurs on?

Any page with the nav bar.

* What happens?

The Log Out menu item is not visible/clickable (still accessible via keyboard tabbing).

* To whom does it occur (all users, curators, superusers)?

Any logged-in user.

* What did you expect to happen?

For Log Out to be visible.

**Which version of Dataverse are you using?**

5.6 - Harvard DV

**Screenshots:**

<img width="1279" alt="Screen Shot 2021-10-07 at 12 11 17 PM" src="https://user-images.githubusercontent.com/6903515/136424026-9771690d-5d54-40db-a892-23af0be5af67.png">

**Comment by pdurbin** [comment]
Log out seems to be visible with the [new React frontend](https://github.com/IQSS/dataverse-frontend): 

<img width="1560" alt="Screenshot 2023-10-08 at 11 16 06 AM" src="https://github.com/IQSS/dataverse/assets/21006/ac44cdf7-de96-41e2-94ef-be30665ad2b5">

(Yes some overlap elsewhere.)

I used https://support.mozilla.org/en-US/questions/1006625 to figure out 200% zoom on Firefox.

Closing.
